packages/opentelemetry-resources/test/Resource.test.ts

```diff
@@ -287,6 +287,116 @@ describe('Resource', () => {
     });
   });
 
+  describe('schema URL support', () => {
+    it('should create resource with schema URL', () => {
+      const schemaUrl = 'https://example.com/schema';
+      const resource = resourceFromAttributes({ attr: 'value' }, schemaUrl);
+
+      assert.strictEqual(resource.getSchemaUrl?.(), schemaUrl);
+    });
+
+    it('should create resource without schema URL', () => {
+      const resource = resourceFromAttributes({ attr: 'value' });
+
+      assert.strictEqual(resource.getSchemaUrl?.(), undefined);
+    });
+
+    it('should merge resources with schema URL priority given to other resource', () => {
+      const resource1 = resourceFromAttributes(
+        { attr1: 'value1' },
+        'https://schema1.com'
+      );
+      const resource2 = resourceFromAttributes(
+        { attr2: 'value2' },
+        'https://schema2.com'
+      );
+
+      const mergedResource = resource1.merge(resource2);
+
+      assert.strictEqual(
+        mergedResource.getSchemaUrl?.(),
+        'https://schema2.com'
+      );
+    });
```

Comment:
I think this is not what should happen - IIUC according to the spec, this is a merge error and should result in an undefined schema url. 

https://opentelemetry.io/docs/specs/otel/resource/sdk/#merge

Reply:
thanks for doing the research! I brought this up with  with @open-telemetry/javascript-maintainers today, we'll go with what java, go and php are doing 🙂